# Request fields
int8   interaction_id
Trajectory observed_trajectory
uint32 num_samples
---
# Response fields
Trajectory generated_trajectory
